src/FluentResults/Results/Result.cs

```diff
@@ -90,8 +90,8 @@ public override string ToString()
         {
             return result.ToResult<TValue>();
         }
-
-        public static implicit operator Result(Result<TValue> result)
+        
+        public static implicit operator Result<TValue>(TValue result)
```

Comment:
I like the removal of the implicit operator Result(Result<TValue> result). Jep, its dangerous but I can't remove directly - because of compatibility/breaking changes reasons. My suggestion is to mark this implicit operator in the next version as deprecated and remove it in the next major version.

Reply:
I like the idea with the tag, I'll fix it